src/components/modal/questionnaireModal.js

```diff
@@ -372,11 +381,12 @@ class QuestionnaireModal extends Component {
 					:
 					(
 						<View>
+							{/* repeat: true */}
```

Comment:
Shouldn't this comment/hint be "repeats: false"?

Reply:
Yes. It should be "repeats: false".

Reply:
fair